app.js

```diff
@@ -11,60 +11,80 @@ const catchphraseInput = document.getElementById('catchphrase-input');
 const catchphraseButton = document.getElementById('catchphrase-button');
 
 // set state for how many times the user changes the head, middle, and bottom
+let headCount = 0;
+let bodyCount = 0;
+let bottomCount = 0;
+
 // set state for all of the character's catchphrases
+const catchphrases = [];
 
 headDropdown.addEventListener('change', () => {
     // get the value of the head dropdown
-
-    // increment the head change count state
+    const value = 'assets/' + headDropdown.value + '-head.png';
     
+    // increment the head change count state
+    headCount++;
     // update the dom for the head (use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
-
+    headEl.style.backgroundImage = `url("${value}")`;
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
 
 middleDropdown.addEventListener('change', () => {
     // get the value of the middle dropdown
-
+    const value = 'assets/' + middleDropdown.value + '-middle.png';
     // increment the middle change count state
+    bodyCount++;
     
     // update the dom for the middle (NOTE: use style.backgroundImage on the middleEl div instead of trying to set the .src -- it's NOT an img tag!)
+    middleEl.style.backgroundImage = `url("${value}")`;
 
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
 
 bottomDropdown.addEventListener('change', () => {
     // get the value of the bottom dropdown
-
+    const value = 'assets/' + bottomDropdown.value + '-pants.png';
     // increment the bottom change count state
+    bottomCount++;
     
     // update the dom for the bottom (NOTE use style.backgroundImage on the bottomEl div instead of trying to set the .src -- it's NOT an img tag!)
+    bottomEl.style.backgroundImage = `url("${value}")`;
 
     // update the stats to show the new count (call displayStats() to do this work)
+    displayStats();
 });
 
 catchphraseButton.addEventListener('click', () => {
     // get the value of the catchphrase input
-    
     // push the new catchphrase to the catchphrase array in state
+    catchphrases.push(catchphraseInput.value);
 
     // clear out the form input's value so it's empty to the user
-   
+    catchphrasesEl.innerHTML = '';
     // update the dom to show the new catchphrases (refactor to/call displayCatchphrases to do this work)
-
+    displayCatchphrases();
 });
 
 function displayStats() {
     // text content of the reportEl to tell the user how many times they've changed each piece of the state
+    reportEl.innerText = 'You have changed the head ' + headCount + ' times, the body ' + bodyCount + ' times, and the pants ' 
+    + bottomCount + ' times. And nobody can forget your characters classic catchphrases:';
 }
 
 function displayCatchphrases() {
     // clear out the DOM for the currently displayed catchphrases
 
```

Comment:
I think per my above comment, this DOM clear doesn't have to happen because you cleared it up there. But, once you fix that one you'll want to do that catchphrasesEl clear here otherwise it'll break your functionality.